village/src/components/Smurfs.js

```diff
@@ -1,22 +1,38 @@
 import React, { Component } from 'react';
+import axios from "axios";
 
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
+
+  deleteItem = (event, id) => {
+    event.preventDefault();
+    axios
+      .delete(`http://localhost:3333/smurfs/${id}`)
+      .then(response => {
+        this.setState({ smurfs: response.data });
+        window.location.reload();
```

Comment:
same problem here... need to pass a method from App that updates its own state and call that method off props here